.fossa.yml

```diff
@@ -28,13 +28,15 @@ analyze:
     type: npm
     target: superset-frontend
     path: superset-frontend
-  - name: docs
+  - name: base
     type: pip
-    target: docs
-    path: docs
-  - name: .
+    target: .
+    path: .
+    options:
+      requirements: ./requirements/base.txt
+  - name: docker
```

Comment:
We don't need this check, as the Docker requirements are only used for the "dev" image (which is used in docker-compose)

Reply:
So, it looks like the "lean" Docker image is using `requirements/local.txt`? This is confusing.

Reply:
Yep, lean gets "local" and dev gets "docker". When I did this, "local" already existed, so I reused it. I added "docker.txt" in order to append a few more deps for docker-compose.

Reply:
Ok, removed it for now. May need some reworking after https://github.com/apache/incubator-superset/pull/10875 settles.